ui.xml

```diff
@@ -246,5 +246,7 @@
         <Button onClick="toggleBlightUI" textAlignment="MiddleCenter">Toggle Blight UI</Button>
         <Button onClick="toggleScoreUI" textAlignment="MiddleCenter">Toggle Score</Button>
         <Button onClick="togglePowerDrawUI" textAlignment="MiddleCenter">Toggle Power Draw UI</Button>
+        <Button onClick="togglePlayerControls" textAlignment="MiddleCenter" tooltip="Toggles the visibility of the buttons to change player seats or colors.  These are automatically hidden when the game starts, but can be turned back on if desired.">Toggle Seat Controls</Button>
```

Comment:
these tooltips should include some line breaks, and most likely want them to appear on the left side

Reply:
I did not see a way to add linebreaks to a tooltip added via the XML UI.  They now are on the left side though.

Reply:
I've added line breaks to the Setup UI tooltips so I'd check out that XML code